item.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+
+# Main super class
+class Item
+  attr_accessor :id, :publish_date, :archived
+
+  def initialize(publish_date)
+    @id = Random.rand(1..1000)
+    @publish_date = Date.parse(publish_date)
+    @archived = false
```

Comment:
- [ ]  There are a few more properties to be declared in the Item class, based on this requirement --> 
`All Item class properties visible in the diagram should be defined and set up in the constructor method. Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.`

![image](https://user-images.githubusercontent.com/8858461/191174110-dc1a1950-4437-4430-a930-ebf07346dc7e.png)

Reply:
Hey @c3c1l1a thanks for your review. I'd appreciate your input. Added these attributes inside the block outside initializing attributes to allow adding these options from child classes `@attr << self`